devnotes/isteps/8_nest_chiplets/1_host_slave_sbe_config.md

```diff
@@ -18,7 +43,8 @@ registers used in this file are described best here `src/include/usr/initservice
 ```python
 # SCRATCH_REGISTER_X is TP.TPVSB.FSI.W.FSI_MAILBOX.FSXCOMP.FSXLOG.SCRATCH_REGISTER_X
 def p9_setup_sbe_config(i_target):
-    # Registers are accessed by SCOM or FSI depending on whether the code is being executed by the MASTER_CHIP
+    # Registers are accessed by SCOM or CFAM/FSI depending on whether the target is
+    # the MASTER_CHIP: master accessed via SCOM and others via CFAM/FSI
```

Comment:
I think this should be `XSCOM` to be precise, unless I misunderstand what CFAM/FSI does.